metricsaggregator/src/main/java/io/dockstore/metricsaggregator/helper/ExecutionCountsAthenaAggregator.java

```diff
@@ -36,7 +35,7 @@
  */
 public abstract class ExecutionCountsAthenaAggregator extends RunExecutionAthenaAggregator<TimeSeriesMetric> {
 
-    private static final int ZONE_HOUR_OFFSET = -4;  // Always aggregate with an Eastern DST time offset, so that all bin boundaries align with Eastern DST midnight and don't shift depending upon where the aggregator is run and/or whether Daylight Savings Time is currently in effect (or not).
+    private static final int ZONE_HOUR_OFFSET = -4;  // Always aggregate with an US Eastern DST time offset, so that all bin boundaries align with US Eastern DST midnight and don't shift depending upon where the aggregator is run and/or whether Daylight Savings Time is currently in effect (or not).
```

Comment:
Think it's the same Eastern time zone for both of us 😁 
https://en.wikipedia.org/wiki/Eastern_Time_Zone